inductiva/scenarios.py

```diff
@@ -47,6 +49,13 @@ def get_config_filename(self, simulator: Simulator):
         """To be implemented in subclasses."""
         pass
 
+    def read_commands_from_file(self, template_dir):
```

Comment:
This should probably accept the full path to the commands file, and not a template dir and search for "commands.json". This can be tacked in a new PR.

Reply:
I've already changed it to a path but I agree with doing the verification.